figures/generate-figures.sh

```diff
@@ -77,14 +82,8 @@ bash ${analyses_dir}/oncoprint-landscape/run-oncoprint.sh
 # Will create two plots - primary only and "primary plus" samples
 filenames=(primary_only primary-plus)
 
-for filename in "${filenames[@]}"; do
-
-  ## Run the `oncoprint-landscape` figure assembly script
-  Rscript --vanilla scripts/oncoprint-landscape.R \
-    --lead_filename ${filename} \
-    --png_name pngs/${filename}_oncoprint_landscape.png
-
-done
+# Create single panel PDFs and legends
+Rscript --vanilla figures/figures/scripts/fig2-oncoprint-landscape.R
```

Comment:
```suggestion
Rscript --vanilla scripts/fig2-oncoprint-landscape.R
```

Reply:
Oops! That's super weird, thanks for catching it!